Raven.NET.Core/Configuration/RavenSettings.cs

```diff
@@ -0,0 +1,8 @@
+namespace Raven.NET.Core.Configuration
+{
+    public class RavenSettings
```

Comment:
Maybe add a todo comment to change those for valid settings in future, now its just dummy data mock right?

Reply:
I'll add this. Even thou rider was shouting at me to remove todo comment